pkg/deploy/apply.go

```diff
@@ -162,6 +163,35 @@ func handleResourceCompletionEvent(res resourceutil.Resource, event *watch.Event
 			return true, nil
 		}
 
+		return false, nil
+	case "ExternalSecret":
+		if event == nil || event.Type != watch.Modified {
+			return false, nil
+		}
+
+		var extsecFromRes, extsecFromEvent externalsecretsv1beta1.ExternalSecret
+		if err := runtime.DefaultUnstructuredConverter.FromUnstructured(res.Object.Object, &extsecFromRes); err != nil {
+			return false, err
+		}
+
+		u, ok := event.Object.(*unstructured.Unstructured)
+		if !ok {
+			msg := "Cannot convert object from event into unstructured while handling ExternalSecret events"
+			return false, errors.New(msg)
+		}
+		if err := runtime.DefaultUnstructuredConverter.FromUnstructured(u.Object, &extsecFromEvent); err != nil {
+			return false, err
+		}
+
+		if extsecFromEvent.Name != extsecFromRes.Name {
+			return false, nil
+		}
+
+		if refreshedAt := extsecFromEvent.Status.RefreshTime; refreshedAt.Time.After(startTime) {
+			fmt.Println("ExternalSecret completed:", extsecFromEvent.Name)
```

Comment:
```suggestion
			fmt.Println("ExternalSecret completed: ", extsecFromEvent.Name)
```

Reply:
This space is not needed since `Println` automatically joins parameter with spaces